code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -172,6 +172,12 @@ Credit where due:
 		return TRUE
 	return FALSE
 
+/datum/game_mode/clockwork_cult/check_finished()
+	var/obj/structure/destructible/clockwork/massive/celestial_gateway/G = locate() in GLOB.all_clockwork_objects
```

Comment:
var/obj/structure/destructible/clockwork/massive/celestial_gateway/G = GLOB.ark_of_the_clockwork_justiciar

Reply:
Why doesn't the proc below this use it?

Reply:
I added the GLOB to the other completion proc, please add +1 GBP for this PR